tools/cellxgene_census_builder/src/cellxgene_census_builder/__main__.py

```diff
@@ -38,31 +38,60 @@ def main() -> int:
     # Process initialization/setup must be done early. NOTE: do NOT log before this line!
     process_init(build_args)
 
-    # Return process exit code (or raise, which exits with a code of `1`)
-    return do_build(build_args, skip_completed_steps=cli_args.test_resume)
+    # List of available commands for the builder
+    commands = {
```

Comment:
I recommend adding a top-level command for old-release-cleanup. Otherwise, the only way it can occur is if you do a full build.